templates/plugin/test.yml.erb

```diff
@@ -0,0 +1 @@
+../test.yml.erb
```

Comment:
It fails on the symlink. Should I just copy the template?

Reply:
Yes :+1: